stories/core_components/GeoPushpinChart.tsx

```diff
@@ -196,7 +199,21 @@ storiesOf("Core components/GeoPushpinChart", module)
                 config,
             }),
         );
-    });
+    })
+    .add("with drillable items", () =>
+        screenshotWrap(
```

Comment:
does geochart with turned on drilling on some item looks different than without? Does it need screenshot?

Reply:
No it doesn’t.
`screenshotWrap` has been removed